src/MUIDataTable.js

```diff
@@ -374,6 +374,8 @@ class MUIDataTable extends React.Component {
     return { columns: columnData, filterData, filterList };
   };
 
+  leaf = (obj, path) => (path.split('.').reduce((value,el) => value[el], obj));
```

Comment:
A very elegant solution, but we need to be forgiving of missing data, as we are with other cells at the moment. If we can't find the object property specified in the path, we should make sure to set `undefined` instead of cause a breaking error.

Reply:
I have made the requested change, but I can't make heads or tails of how to test it properly. If you can offer some guidance on where the test(s) would live I can give it a shot, but you can see from the modified example that I spot checked with invalid attributes and invalid sub-attributes.